client/login/magic-login/index.jsx

```diff
@@ -249,24 +249,28 @@ class MagicLogin extends Component {
 		);
 	}
 
-	renderGravatarMagicLogin() {
+	renderGravPoweredMagicLogin() {
 		const { oauth2Client, translate, locale, query } = this.props;
 
+		const isWPJobManager = isWPJobManagerOAuth2Client( oauth2Client );
+		const headerText = isWPJobManager
+			? translate( 'Welcome to WP Job Manager' )
```

Comment:
WDYT about using "Login to Gravatar" for Gravatar's client ID, but use a more generic "Welcome to {name}" for the rest of gravatar-powered client IDs?